src/Serilog.Sinks.Exceptionless/Sinks/Exceptionless/ExceptionlessSink.cs

```diff
@@ -66,7 +67,13 @@ public class ExceptionlessSink : ILogEventSink {
         public ExceptionlessSink(Func<EventBuilder, EventBuilder> additionalOperation = null, bool includeProperties = true, ExceptionlessClient client = null) {
             _additionalOperation = additionalOperation;
             _includeProperties = includeProperties;
-            _client = client ?? ExceptionlessClient.Default;
+            if (client != null) {
+                _client = client;
+            }
+            else {
+                _client = ExceptionlessClient.Default;
+                _client.Configuration.UseLogger(new SelfLogLogger());
```

Comment:
There isn't but the default logger will be https://github.com/exceptionless/Exceptionless.Net/blob/f0494c8015fe8ba76cdb11b9b017c6c1d68f8749/src/Exceptionless/Logging/NullExceptionlessLog.cs